### ISSUE TITLE
Database access in tests

### ISSUE THREAD

**Issue by macobo** [amjith/pgcli#68]
It's currently quite hard to make sure that no new regressions are introduced while adding new functionality or fixing bugs.

I would propose adding support for a tests that have access to a running postgres instance and can use it to run commands against. If postgres is not running, such tests would be skipped.

Travis [does have support for postgres](http://docs.travis-ci.com/user/using-postgresql/) and with some experimentation I'm sure turning it on in wouldn't be a problem.

@amjith, how do you feel about this? Would it be something that agrees with your plans for the project?

**Comment by amjith** [comment]
Yes. That is indeed in the grand plan. https://github.com/amjith/pgcli/blob/master/tests/test_plan.wiki

Just haven't gotten enough hours in a day. :)

**Comment by macobo** [comment]
Woot. We'll see, maybe I'll throw something together. :)

Cheers!
